src/compiler/utilities.ts

```diff
@@ -1281,402 +1282,400 @@ export function getInternalEmitFlags(node: Node): InternalEmitFlags {
+        ],
+    })),
+    NumberFormat: new Map(Object.entries({
+        es2018: [
+            "formatToParts",
+        ],
+    })),
+    SymbolConstructor: new Map(Object.entries({
+        es2020: [
+            "matchAll",
+        ],
+    })),
+    DataView: new Map(Object.entries({
+        es2020: [
+            "setBigInt64",
+            "setBigUint64",
+            "getBigInt64",
+            "getBigUint64",
+        ],
+    })),
+    BigInt: new Map(Object.entries({
+        es2020: emptyArray,
+    })),
+    RelativeTimeFormat: new Map(Object.entries({
+        es2020: [
+            "format",
+            "formatToParts",
+            "resolvedOptions",
+        ],
+    })),
+    Int8Array: new Map(Object.entries({
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    Uint8Array: new Map(Object.entries({
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    Uint8ClampedArray: new Map(Object.entries({
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    Int16Array: new Map(Object.entries({
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    Uint16Array: new Map(Object.entries({
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    Int32Array: new Map(Object.entries({
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    Uint32Array: new Map(Object.entries({
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    Float32Array: new Map(Object.entries({
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    Float64Array: new Map(Object.entries({
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    BigInt64Array: new Map(Object.entries({
+        es2020: emptyArray,
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    BigUint64Array: new Map(Object.entries({
+        es2020: emptyArray,
+        es2022: [
+            "at",
+        ],
+        es2023: [
+            "findLastIndex",
+            "findLast",
+        ],
+    })),
+    Error: new Map(Object.entries({
+        es2022: [
+            "cause",
+        ],
+    })),
+})) as ScriptTargetFeatures);

```

Comment:
Instead of casting, you probably just want to write `(): ScriptTargetFeatures => new Map()`

Reply:
ok, I removed the casting